improve: Make it easier for other people to follow package-maker (#52)
gap-packages/PackageMaker: README.md

        git clone https://github.com/gap-packages/PackageMaker

> Or, if you're familiar with `PackageManager`, you may want to install `PackageMaker` by `PackageManager`, which is easier to use and automatically installs packages for you by leveraging [PackageManager](https://github.com/gap-packages/PackageManager).

Reviewer: ```suggestion
Or, you may want to install `PackageMaker` using the [PackageManager](https://github.com/gap-packages/PackageManager) GAP package by typing `LoadPackage("PackageManager"); InstallPackage("PackageMaker");` in GAP.
```

> Or, you may want to install `PackageMaker` using the [PackageManager](https://github.com/gap-packages/PackageManager) GAP package by typing `LoadPackage("PackageManager"); InstallPackage("PackageMaker");` in GAP.
> 
>         InstallPackage("PackageMaker");

Reviewer 2: This won't work right now because PackageManager won't know where to find this package. Perhaps like this, at least for now:
```suggestion
Alternatively you could install `PackageMaker` using the [PackageManager](https://github.com/gap-packages/PackageManager) GAP package by entering these commands in GAP:

        LoadPackage("PackageManager");
        InstallPackage("https://github.com/gap-packages/PackageMaker");
```

Author: @fingolfin Why isn't it working? I have been using the package manager to install packages recently, and it works well. As Michel mentioned in his talk yesterday, he is currently fetching data from the `csv`, but he plans to change the structure and fetch all the data in one go using a caching mechanism from a new `json` file, which already exists.

2. Start GAP, load the `PackageMaker` package:

        LoadPackage("PackageMaker");

3. Run the package wizard:

        PackageWizard();

4. Answer the questions about your new package. Afterwards, `PackageMaker`
   creates a new directory for the new package and populates it with all the
   files needed for a basic package.

5. Move the newly created package directory to a suitable place.

> > Note: The `PackageWizard` function generates your new package in the current directory, not automatically in the `pkg` directory. You will need to manually move the generated directory to the `pkg` folder in your GAP installation.

Reviewer 2: ```suggestion
> Note: The `PackageWizard` function generates your new package in the current directory, not automatically in the `pkg` directory. You will need to manually move the generated directory to the `pkg` folder in your GAP installation.
Or just start GAP inside the `pkg` directory.
```

... but perhaps it would be better to instead change the part were we say "Start GAP" to instead say something like: "Change into the directory where you want to create your new package. For example the `pkg` directory of your GAP installation, or the directory `~/.gap/pkg` (you may have to create it first) -- then GAP will automatically find your package after it has been created".

That could then be a step of its own and we could drop this comment and also the "Move the newly created package directory ..." step...

Thoughts?

Author: Yes good idea, I created a new issue.
https://github.com/gap-packages/PackageMaker/issues/58
